com.unity.netcode.adapter.utp/Runtime/UnityTransport.cs

```diff
@@ -161,6 +161,7 @@ public struct ConnectionAddressData
         private NetworkPipeline m_UnreliableFragmentedPipeline;
         private NetworkPipeline m_UnreliableSequencedFragmentedPipeline;
         private NetworkPipeline m_ReliableSequencedFragmentedPipeline;
+        private const byte k_PipelineCount = 3; //Represented the number of existing pipeline
```

Comment:
I think it would be best to have an enum for this, rather than referring to these stages using raw numbers in the switch statement later. Something like:
```
enum PipelineType
{
  UnreliableFragmented,
  UnreliableSequenceFragmented,
  ReliableSequenceFragmented,
  Count,
}
```
This would replace count with `(int)PipelineType.Count`

Reply:
Sure. I just wanted something simple in case things pipeline would change but you're right.